### DISCUSSION TITLE
0.10 Rocks!

### DISCUSSION THREAD

**Discussion by lkoepsel** [bloombloombloom/Bloom#45] [General]
Hi Nav,

Thank you for advising me as to the release of 0.10. In my preliminary testing, it works great! A couple of points:
1) Thanks for adding an Arch Linux version to your releases. It made my update extremely easy!
2) My very simple test was to load a program (Blink, duh) then edit and change the time. Incredibly simple!
3) I'm swamped with issues right now, however I'll update my bloom pages with the latest as this is now a full circle tool. It is so easy to edit/reload, its fantastic!!

Thank you, again!
Lief

**Comment by navnavnav** [comment]
Hi Lief,

Thanks for the feedback! It's great to hear that v0.10 is working as intended. Feel free to send over any links to your updated articles - I will be happy to go over them.

Thanks!

**Comment by lkoepsel** [comment]
I've updated one entry:
[Developing in C for the ATmega328: Setup Bloom and gdb for Hardware Debug](https://wellys.com/posts/avr_c_gdb_bloomsetup/
)

I'll continue to work on them and let you know when they've been updated. 

A minor note, at one point in time today, I had an error which ended with "...debugWIRE physical error". I simply cycled power and the problem went away. You might want to think about adding a comment in your error messages as to "Please cycle power to attempt to fix."

**Comment by lkoepsel** [comment]
Not a Bloom issue, however, you might have some insight. I'm unable to use avrdude with the snap board. Here's the issue discussion on github/avrdude:

[ opening programmer "snap_isp" on port "usb" failed #1017 ](https://github.com/avrdudes/avrdude/issues/1017)

Any thoughts or comments would be deeply appreciated.

**Comment by navnavnav** [comment]
> Not a Bloom issue, however, you might have some insight. I'm unable to use avrdude with the snap board. Here's the issue discussion on github/avrdude:

Hi Lief,

I am able to replicate this on my Manjaro VM, but only when I install avrdude via the Arch Linux package manager (`pacman -S avrdude`). I believe there is something wrong with that particular build, but I'm not sure what. When I replace the binary with one I built from the latest source, it works fine.

I'm not sure what version of the source that Arch Linux package was built from, but given that the repo's main branch was updated just 4 days ago, the Arch Linux package is probably built from an older version of the source, and this bug has since been fixed in a later commit.

Might be an idea to ask the Arch Linux package maintainers to update the package. In the meantime, you can either build avrdude yourself, from the latest source, or ~~use the binary I built: https://bloom-public-bucket.s3.eu-west-1.amazonaws.com/misc/avrdude (BTW that link will expire in 5 days).~~

Actually, instead of using the binary I built, you can download the latest GitHub artifact from the avrdude repo: https://github.com/avrdudes/avrdude/suites/7134298959/artifacts/283330607 (via https://github.com/avrdudes/avrdude/actions/runs/2579353870). That one was built just 4 days ago, and I can confirm it works on my Manjaro VM. But note that you'll also have to copy the avrdude.conf (provided in the artifact) to `/usr/local/etc`, as the expected location of that file seems to have changed (it was initially `/etc/`).

Let me know how that goes

Thanks,

**Comment by lkoepsel** [comment]
Yes! Many thanks, I was going down the path of "maybe its Arch Linux...when my Ubuntu install crashed..." :)

I am seeing painfully slow programming times (~17s for 1046 bytes), I attempted to set the baud rate to 115200 and 38400 to no avail.

I'm going to let it go as I can use Bloom to program. :)

Many thanks,
Lief
